### ISSUE TITLE
[2.x] [Bug] Not Detecting The Latest Addon Release Version

### ISSUE THREAD

**Issue by ManagerWoW** [WowUp/WowUp#306]
WowUp Version: 2.0.0-beta.1
OS: Windows

Hello! Out of my 73 addons everything is working perfectly except for 1 addon: [idTip](https://www.curseforge.com/wow/addons/idtip)

WowUp thinks the latest Release Version is 8.1.4 with a Game Version of 1.13.2, even though there are multiple newer release versions and I am on Retail. [Picture](http://manager.heroicstrike.org/xxbAprz2.png)

I have tried reinstalling and uninstalling idTip through WowUp but it always thinks 8.1.4 is the newest version. Searching for it through "Get Addons" in WowUp also shows the 8.1.4 version.

**Comment by ManagerWoW** [comment]
Looks like Geist also has this issue: https://www.curseforge.com/wow/addons/geist/files

Get Addons detects the 8.1.0 version as the newest, not 9.0.1: [Picture](http://manager.heroicstrike.org/STdjneju.png)

Geist is also not being detected as installed, even after doing Re-Scan Folders multiple times.

**Comment by jliddev** [comment]
These ones are interesting. they are both tagged "gameVersionFlavor": "wow_classic" which is why you're seeing issues detecting them for retail. However, they have two game versions listed

**Comment by tupper7** [comment]
I am having the same issue with idTip, I tried adding it via link in 2.0.0-beta.2 but there is an error.

https://github.com/silverwind/idTip
https://www.wowinterface.com/downloads/fileinfo.php?id=17033

**Comment by jliddev** [comment]
@tupper7 this seems like a different issue try https://www.wowinterface.com/downloads/info17033-idTip.html

**Comment by tupper7** [comment]
Thanks, that worked.  wowinterface gives me the correct version of idTip unlike curse.

**Comment by cremor** [comment]
ElkExperience (https://www.curseforge.com/wow/addons/elkexperience/files) is another addon with a similar problem. The CurseForge website shows "1.0.0 - r9" as the latest (beta) version, but WowUp only shows version "1.0.0 - r8" (also a beta).

**Comment by cremor** [comment]
@jliddev Could you please comment on why you closed this issue? Is it fixed? If yes, in which version of WowUp? v2.1.2 is still not showing version "1.0.0 - r9" of ElkExperience.

**Comment by jliddev** [comment]
The issue with this addon is that the file is tagged with mulitple game versions 
![image](https://user-images.githubusercontent.com/20467484/110809363-6e05d980-824a-11eb-90bd-85c1df34e3a3.png)


There is another ticket somewhere that deals with this specific problem.

TLDR that release is tagged as classic by CF API.
![image](https://user-images.githubusercontent.com/20467484/110809468-85dd5d80-824a-11eb-9025-b687c6b41b51.png)

**Comment by cremor** [comment]
Is this other ticket still open? Because I can only find closed issues about that problem.
In other words: Might we see a correct handling (from a user PoV) of such addons in WowUp some day?

**Comment by linaori** [comment]
There's no way WowUp can correctly detect this from a user PoV, because curse is feeding the wrong information.

**Comment by cremor** [comment]
Why can't WowUp ignore the wrong "gameVersionFlavor" and see that "gameVersion" contains both a classic and a retail WoW version?

**Comment by linaori** [comment]
Because as you can see, this game version is set to 8.2, which means it would be for BfA, not for SL and thus not match either. Especially when TBC comes out, you'll have 1.x, 2.x and 9.x. 1 mistake from the author and you have the wrong version. It happens occasionally when the author adds the wrong version here, which is also why wowup uses the toc version whenever available.

**Comment by cremor** [comment]
I use multiple addons meant for older WoW versions, one even for 6.2.0. Those addons can still work fine today. Why not draw the line somewhere, e.g. treat 1.x as classic, 2.x as TBC and everything else as retail?

**Comment by linaori** [comment]
If the addon is still maintained, ask the author to fix it on curseforge. If the addon is no longer maintained, you can install it manually as there won't be any updates, so it will be a one time thing.

**Comment by cremor** [comment]
As I've written in my first comment of this issue, the problem is how to find addons that are affected by this. I have over 100 addons and as far as I can see the only way would be to check all of them manually.
